lib/beaker-hostgenerator/cli.rb

```diff
@@ -121,7 +121,7 @@ def initialize(argv = ARGV.dup)
         raise BeakerHostGenerator::Exceptions::SafeEarlyExit
       else
         # Tokenizing the config definition for great justice
-        @tokens = argv[0].split('-')
+        @tokens = __tokenize_input(argv[0])
```

Comment:
I don't either, was just sticking to the convention.  Although I notice that we didn't have the `__method` convention in this class, oops!  I'll remove that part and throw in a `private`